modules/tiled-gallery/tiled-gallery/tiled-gallery-item.php

```diff
@@ -21,8 +21,7 @@ public function __construct( $attachment_image, $needs_attachment_link, $graysca
 		$this->orig_file = wp_get_attachment_url( $this->image->ID );
 		$this->link = $needs_attachment_link ? get_attachment_link( $this->image->ID ) : $this->orig_file;
 
-		$this->img_src = add_query_arg( array( 'w' => $this->image->width, 'h' => $this->image->height, 'crop' => true ), $this->orig_file );
-
+		$this->img_src = jetpack_photon_url( $this->orig_file, array( 'resize' => sprintf( '%s,%s', $this->image->width, $this->image->height ) ) );
```

Comment:
If you're using `sprintf` here, then those dimensions should probably be cast to decimal/numbers (`%d`), rather than strings (`%s`).